tagstudio/src/core/library.py

```diff
@@ -864,7 +868,15 @@ def refresh_dir(self) -> Generator:
                     and "tagstudio_thumbs" not in f.parts
                     and not f.is_dir()
                 ):
-                    if f.suffix not in self.ignored_extensions:
+                    if f.suffix not in self.ext_list and self.is_exclude_list:
+                        self.dir_file_count += 1
+                        file = f.relative_to(self.library_dir)
+                        try:
+                            _ = self.filename_to_entry_id_map[file]
+                        except KeyError:
+                            # print(file)
+                            self.files_not_in_library.append(file)
```

Comment:
last nitpick, sorry. I know this is copy of the existing code, but let's make a bit better:
```suggestion
                        if file not in self.filename_to_entry_id_map:
                            self.files_not_in_library.append(file)
```

Reply:
All good.